### PR TITLE
UDUNITS clarifications from workshop housekeeping 2022

### PR DESCRIPTION
**Pull request by davidhassell**
See issue #385  for discussion of these changes.

# Release checklist
- [x] Authors updated in `cf-conventions.adoc`?
- [x] Next version in `cf-conventions.adoc` up to date? Versioning inspired by [SemVer](https://semver.org).
- [ ] `history.adoc` up to date?
- [x] Conformance document up-to-date?

# For maintainers
After the merge remember to delete the source branch.
Tags are set at the conclusion of the annual meeting; until then `master` always is a draft for the next version.

### CODE REVIEW

File: ch04.adoc [hunk 1]
```diff
@@ -55,11 +51,11 @@ float lat(lat) ;
 
 ====
 
-Application writers should note that the Udunits package does not recognize the directionality implied by the "north" part of the unit specification.
+Application writers should note that the UDUNITS package does not recognize the directionality implied by the "north" part of the unit specification.
 It only recognizes its size, i.e., 1 degree is defined to be pi/180 radians.
 Hence, determination that a coordinate is a latitude type should be done via a string match between the given unit and one of the acceptable forms of **`degrees_north`**.
 
-Optionally, the latitude type may be indicated additionally by providing the **`standard_name`** attribute with the value **`latitude`**, and/or the **`axis`** attribute with the value **`Y`**.
+It is strongly recommended that the latitude type is also indicated by providing the **`standard_name`** attribute with the value **`latitude`**, and/or the **`axis`** attribute with the value **`Y`**.
```
Comment (davidhassell) [inline ch04.adoc:58]: I think this change (and the similar change to the longitude section), is substantive and also requires an update to the conformance document. So I shall revert this line back to the original (which is nice and easy now that one sentence per line!)

File: ch04.adoc [hunk 2]
```diff
@@ -91,11 +87,11 @@ float lon(lon) ;
 
 ====
 
-Application writers should note that the Udunits package has limited recognition of the directionality implied by the "east" part of the unit specification.
+Application writers should note that the UDUNITS package has limited recognition of the directionality implied by the "east" part of the unit specification.
 It defines **`degrees_east`** to be pi/180 radians, and hence equivalent to **`degrees_north`**.
 We recommend the determination that a coordinate is a longitude type should be done via a string match between the given unit and one of the acceptable forms of **`degrees_east`**.
 
-Optionally, the longitude type may be indicated additionally by providing the **`standard_name`** attribute with the value **`longitude`**, and/or the **`axis`** attribute with the value **`X`**.
+It is strongly recommended that the longitude type is also indicated by providing the **`standard_name`** attribute with the value **`longitude`**, and/or the **`axis`** attribute with the value **`X`**.
```
Comment (davidhassell) [inline ch04.adoc:94]: I think this change (and the similar change to the latitude section), is substantive and also requires an update to the conformance document. So I shall revert this line back to the original.

**Review by davidhassell — COMMENTED** [review]
Removal of new recommadations

File: ch04.adoc (hunk 1 above)
Comment (larsbarring) [reply ch04.adoc:58]: I agree that it is a substantive change , so yes remove it. The question then is if the change is a good one that should be picked up in a separate discussion issue?

**Comment by erget** [comment]
Looks like @davidhassell you have reviewed and implemented the changes to the PR that you proposed. Thus I have resolved the discussion around them. Are you happy for this to be merged or is this still WIP?

**Review by larsbarring — APPROVED** [review]

**Comment by erget** [comment]
Sorry - these need to be rebased onto the Conventions as I should have waited for the 3w deadline to pass. I'll note that in the comment and start the countdown for 3w.

**Comment by erget** [comment]
Just cherry-picked the commits from this reverted PR onto `main`

**Pull request merged**